packages/polymath-issuer/src/pages/compliance/components/ImportWhitelistModal.js

```diff
@@ -3,13 +3,13 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import {
-  Modal,
+  // Modal,
```

Comment:
👁

Reply:
Resolved

Reply:
Why leaving it as commented?

Reply:
I already fixed this in the latest commit